### ISSUE TITLE
Error removing package

### ISSUE THREAD

**Issue by vidstige** [heartsucker/node-deb#35]
It's me the issue spammer again. Found another small issue

When `sudo apt remove myapp` I get this consistently.

```
Failed to stop myapp.service\x7d.service: Unit myapp.service\x7d.service not loaded.
myapp wasn't even running!
```

The problem seems to be a spurious closing brace at [prerm:12](/heartsucker/node-deb/blob/a90eb19367f28beca12aaee76cd0000ee0273cb/templates/prerm#L12)

**Comment by vidstige** [comment]
Alright, fix in #36

**Comment by heartsucker** [comment]
Fixed by #36
